modules/javafx.web/src/main/java/javafx/scene/web/HTMLEditorSkin.java

```diff
@@ -830,9 +826,7 @@ private Button addButton(ToolBar toolbar, final String iconName, String tooltipT
         button.getStyleClass().add(styleClass);
         toolbar.getItems().add(button);
 
-        @SuppressWarnings("removal")
-        Image icon = AccessController.doPrivileged((PrivilegedAction<Image>) () -> new Image(HTMLEditorSkin.class.getResource(iconName).toString()));
-//        button.setGraphic(new ImageView(icon));
```

Comment:
I see you removed this comment. Do you think there is no value in leaving it?

Reply:
reverted here and below.